Tools/CISupport/ews-build/steps.py

```diff
@@ -4863,18 +4863,19 @@ def run(self):
                 ['git', 'branch', '-f', base_ref, head_ref],
                 ['git', 'checkout', base_ref],
             ]
-        commands.append(['python3', 'Tools/Scripts/git-webkit', 'canonicalize', '-n', '1'])
+        commands.append(['python3', 'Tools/Scripts/git-webkit', 'canonicalize', '-n', '1' if self.rebase_enabled else '3'])
```

Comment:
Is this something we will be removing after moving to pure git world? If so, please add a FIXME. Also, in general, it would be good to add FIXME for code which would need to be removed after moving to pure git.

Reply:
The concept of a no-rebased `canonicalize` will go away. Merge-Queue will still be canonicalizing commits. I think the comment is actually more appropriate would be in #263